caravel/storage/entities.py

```diff
@@ -55,6 +58,7 @@ def keywords(self):
         # Tokenize title and body (ranking them equally)
         words = self.title.split() + self.body.split()
```

Comment:
I mean, it's not broken, really — it just removes a potential source of confusion in the future.